websites/S/sekolahmu/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "id": "999863217557880842",
+    "name": "vintheweirdass"
+  },
+  "service": "sekolahmu",
+  "altnames": ["sekolah.mu", "smm"],
+  "description": {
+    "en": "Unofficial Discord presence of Sekolahmu LMS",
+    "id": "Unofficial status Discord buat LMS Sekolahmu"
+  },
+  "url": "www.sekolah.mu",
+  "version": "1.0.0",
+  "logo": "https://raw.githubusercontent.com/vintheweirdass/sekolahmu-premid-assets/refs/heads/main/images/logo-v2.png",
+  "thumbnail": "https://raw.githubusercontent.com/vintheweirdass/sekolahmu-premid-assets/refs/heads/main/images/logo-v2.png",
```

Comment:
The thumbnail should ideally be a landscape promotional image or screenshot, not the same as the logo